app/facades/merchants_facade.rb

```diff
@@ -0,0 +1,21 @@
+class MerchantsFacade
+  def initialize(id = nil)
+    @id = id
+  end
+
+  def service
+    EngineService.new
+  end
```

Comment:
Similarly here, no need to create a new instance of the Service class -- we dont need to store any state -- we just want to call to a method and have that method call to our service.